CONTRIBUTING.md

```diff
@@ -38,7 +38,7 @@ join the [contributions channel](https://discord.gg/CQueVjk3fc) on EA Hub's disc
 that you'd like to contribute.  
 
 ### Pull Requests  
-To make a pull request, please fork the repo, create a branch on your fork and then make a PR into the main repo.
+To make a pull request, please fork the repo, create a branch on your fork and then make a PR into the main repo. For a detailed tutorial, please see our Contribution Tutorial at the end of this page.
```

Comment:
Thanks for fixing all the language issues! :)

Reply:
Cheers :) Let me know if there are other pages I should check for such things.